smilerating/src/main/java/com/hsalf/smilerating/SmileRating.java

```diff
@@ -36,6 +35,13 @@
     private int mAngryColor = Color.parseColor("#f29a68");
     private int mNormalColor = Color.parseColor("#f2dd68");
```

Comment:
Got your point. I left it out as it is so that what i have added can be easily reflected in code